### DISCUSSION TITLE
Save mesh as .off

### DISCUSSION THREAD

**Discussion by atale480** [pyg-team/pytorch_geometric#6503] [Q&A] [answered]
How can I save mesh as `.off` file or other formats for mesh ?
I'm getting this mesh as output of some operations. I have edge_index as well as x,y,z co-ordinates of mesh.

**Comment by rusty1s** [comment, marked as answer]
The `edge_index` defines the connectivity between pairs of nodes, it is (sadly) non-trivial or not possible to convert this to triangles.

**Reply by atale480** [reply]
Okay. Just one doubt if I'm getting mesh after performing some operations then as triangle connectivity in not changing then if I'm having triangle connectivity information so just by changing node x,y,z in same file will do job right?

Because I'm not sure that vertex number 0 in original Tensor and vertex number 0 in processed Tensor  will be x,y,z co-ordinate of same vertex or of different vertex?

**Reply by rusty1s** [reply]
Yes, that is correct. The vertex numbers should be unchanged.
